src/test/java/cart/repository/MemberRepositoryTest.java

```diff
@@ -0,0 +1,89 @@
+package cart.repository;
+
+import cart.domain.member.Member;
+import cart.domain.member.MemberId;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.util.List;
+
+@SpringBootTest
+public class MemberRepositoryTest {
+	@Autowired
+	MemberRepository memberRepository;
+
+	@Autowired
+	JdbcTemplate jdbcTemplate;
+
+	@BeforeEach
+	void setUp() {
+		jdbcTemplate.execute("SET FOREIGN_KEY_CHECKS = 0");
+		jdbcTemplate.execute("TRUNCATE TABLE members RESTART IDENTITY");
+		jdbcTemplate.execute("SET FOREIGN_KEY_CHECKS = 1");
+	}
+
+	@DisplayName("회원 저장 테스트")
+	@Test
+	void insert() {
+		// given
+		final Member member = new Member("kiara", "email@email", "pw");
+
+		// when
+		MemberId insertId = memberRepository.insert(member);
+
+		// then
+		Assertions.assertThat(new MemberId(1L)).isEqualTo(insertId);
+	}
+
+	@DisplayName("전체 회원 조회 테스트")
+	@Test
+	void findAll() {
+		// given
+		final Member member = new Member("kiara", "email@email", "pw");
+
+		// when
+		memberRepository.insert(member);
+		List<Member> allMembers = memberRepository.findAll();
+
+		// then
+		Assertions.assertThat(1).isEqualTo(allMembers.size());
```

Comment:
Assertions.assertThat(List<T> c) 메서드는 ListAssert 객체를 반환하는데요, 해당 객체에는 hasSize() 라는 메서드가 있습니다 
```suggestion
		Assertions.assertThat(allMembers).hasSize(1);
```

Reply:
좋은 기능이네요..! 하나 배워갑니당